src/WebJobs.Extensions.DocumentDB/Trigger/DocumentDBTriggerConstants.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Extensions.DocumentDB
+{
+    internal static class DocumentDBTriggerConstants
+    {
+        public const string DefaultLeaseCollectionName = "leases";
+
+        public const string TriggerName = "DocumentDBTrigger";
+
+        public const string TriggerDescription = "New changes on collection {0} at {1}";
```

Comment:
Okay, that was just a little "nice-to-have". We can always revisit later if we figure out a way.